doc/index.rst

```diff
@@ -1,28 +1,53 @@
+Pylint documentation
+====================
 
-Pylint User Manual
-==================
+Pylint is a `static code analyser`_ for Python 2 or 3. The latest version supports Python
+3.7.2 and above.
 
-Pylint is a tool that checks for errors in Python code, tries to enforce a
-coding standard and looks for code smells. It can also look for certain type
-errors, it can recommend suggestions about how particular blocks can be
-refactored and can offer you details about the code's complexity.
+Pylint analyses your code without actually running it. It checks for errors, enforces a coding
+standard, looks for `code smells`_, and can make suggestions about how the code could be refactored.
 
+Project that you might want to use alongside pylint include flake8_ (faster and simpler checks
+with very few false positives), mypy_, pyright_ or pyre_ (typing checks), bandit_ (security
+oriented checks), black_ and isort_ (auto-formatting), autoflake_ (automated removal of
+unused import or variable), pyupgrade_ (automated upgrade to newer python syntax) and
+pydocstringformatter_ (automated pep257).
```

Comment:
😄

Reply:
It's your last chance to give this tool a cool name, Daniel.

Reply:
Ship has sailed I think 😄 And I really don't know anything better.